q2cli/tests/test_tools.py

```diff
@@ -247,6 +247,18 @@ def test_export_visualization_to_dir_success_message(self):
                   'directory %s\n' % (self.viz, output_path)
         self.assertEqual(success, result.output)
 
+    def test_export_to_filename_without_path(self):
```

Comment:
Can you make sure the following tests are present (add if they are not):

- export a file-format into a directory (`foo_dir/table.biom`)
- export a directory-format into the current directory (`foo_dir`)
- export a directory-format into a nested (`foo_dir/bar_dir`)

Reply:
I'm not sure I really understand what you're asking here. Is `foo_dir` supposed to be the current working directory in all three cases? For the first case, we want to export a file-format (defined as something inheriting from `_FileFormat`?) into an actual filepath and NOT just a filename? For the second we want to export a directory-format (presumably defined in a similar manner to a file-format) into the current directory? And for the third we want to do the same as the second but export to a subdirectory of the current directory?

Reply:
I have updated the list, for clarity.

>  Is foo_dir supposed to be the current working directory in all three cases?

No, `foo_dir` is a dir within your current working directory.

>  For the first case, we want to export a file-format (defined as something inheriting from _FileFormat?) into an actual filepath and NOT just a filename?

No, we want to export to a directory within our current working directory.

> For the second we want to export a directory-format (presumably defined in a similar manner to a file-format) into the current directory?

:+1: 

>  And for the third we want to do the same as the second but export to a subdirectory of the current directory?

:+1: